charts/scatterCharts/ScatterTransform.ts

```diff
@@ -748,17 +752,14 @@ function cagr(
     if (targetValue.year - indexValue.year === 0) return 0
     else {
         const frac = targetValue[property] / indexValue[property]
-        if (frac < 0)
-            return (
-                -(
-                    Math.pow(-frac, 1 / (targetValue.year - indexValue.year)) -
-                    1
-                ) * 100
-            )
-        else
-            return (
-                (Math.pow(frac, 1 / (targetValue.year - indexValue.year)) - 1) *
-                100
-            )
+        return (
+            Math.sign(frac) *
+            (Math.pow(
+                Math.abs(frac),
```

Comment:
I'm using `Math.sign` and `Math.abs` in here now so we don't need to unnecessarily duplicate this massive term.

Reply:
Cleaner!

Also, another one where at some point we should probably add tests and/or just call a tested library fn.